### PR TITLE
[ENH, MRG] PSD multitaper per taper

### PR DESCRIPTION
**Pull request by mmagnuski**
#### Reference issue
Continuation of my previos PR (#10281).

#### What does this implement/fix?
This time per taper results can be obtained from `psd_array_multitaper` as well.

#### Additional information
This required adding an `output` arg to `psd_array_multitaper`, when `output='complex'` the `psd` array has tapers dimension. I also return `weights` in that case as well, but I'm not sure about this - the weights are generally marginally different from 1, so returning weights might not be needed (and it is not done in the `tfr_array_multitaper`, which does not use any weights, assuming they are all 1). However, weights might be useful at least for testing to check that we can get the same power results from complex output.

#### Remainging tasks
- [x] decide whether to return the weights or not
- [x] add test
- [x] update whats new

### CODE REVIEW

**Comment by mmagnuski** [comment]
Oh, I think I got the normalization wrong, will fix that.

**Comment by larsoner** [comment]
... oh, you fixed the indentation already :)

File: mne/time_frequency/multitaper.py [hunk 1]
```diff
@@ -420,38 +425,52 @@ def psd_array_multitaper(x, sfreq, fmin=0, fmax=np.inf, bandwidth=None,
...
     # Combining/reshaping to original data shape
-    psd.shape = dshape + (-1,)
+    last_dims = (n_freqs,) if output == 'power' else (n_tapers, n_freqs)
+    psd.shape = dshape + last_dims
     if ndim_in == 1:
         psd = psd[0]
-    return psd, freqs
+
+    if output == 'complex':
+        return psd, freqs, weights
```
Comment (larsoner) [inline mne/time_frequency/multitaper.py:471]: I would output the weights in this case. It's easy enough to ignore them if you don't want them

Returns needs to be updated, though

**Comment by mmagnuski** [comment]
The `build_docs` error looks unrelated: `ERROR: Failed building wheel for statsmodels`.

**Comment by larsoner** [comment]
Thanks @mmagnuski !

**Comment by mmagnuski** [comment]
Thanks @larsoner :rocket:!

**Pull request merged**